### PR TITLE
Expose the prefix on the config instance

### PR DESCRIPTION
**Pull request by codazzo**
It can be useful for scripts etc.

### CODE REVIEW

**Comment by spmason** [comment]
:+1: Why the test failure tho?

**Comment by codazzo** [comment]
I think npm had borked. Passing now.

File: test/index_test.js [hunk 1]
```diff
@@ -25,6 +25,15 @@ describe('config', function () {
         assert.equal(config.TEST, 'test');
     });
 
+    it('exposes the prefix', function () {
```
Comment (spmason) [inline test/index_test.js:28]: This looks like 2 tests:
1. exposes the prefix
2. does not allow prefix to be changed

Comment (codazzo) [inline test/index_test.js:28]: Fair point. Will fix.

**Comment by codazzo** [comment]
Looking good now @spmason ?

**Comment by spmason** [comment]
:shipit:

**Pull request merged**